fix backwards-incompatible change on gcsfuse

sirdarckcat opened the pull request.

sirdarckcat committed:
fix backwards-incompatible change on gcsfuse

sirdarckcat committed:
Automated commit: update images.

sroettger commented:
You'll probably have to make another dummy commit to the operator to trigger a rebuild

sirdarckcat committed:
Update Dockerfile

allow caching the type (dir vs file) so that things are a bit faster. this creates weird race conditions (file vs. dir) but that should be fine (tm)

sirdarckcat commented:
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#name-conflicts

sirdarckcat committed:
dummy commit

sirdarckcat committed:
Automated commit: update images.
